containers/Auth/StartUpScreen.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import { ActivityIndicator, View } from 'react-native';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+import { styles } from '../Ideas/Idea.styles';
+import { COLORS } from '../../theme/colors';
+
+export const StartUpScreen = (props) => {
+
+  useEffect(() => {
+    const tryLogin = async () => {
+      const authToken = await AsyncStorage.getItem('authToken');
+      if (!authToken) {
+        props.navigation.navigate('Auth');
+      }
+      else {
```

Comment:
just a little thing, but we should align on the same brace style for if-statements.
I also like the stroustrup brace style. @Rineee @phillimorland @fuzzylogic2000 if you don't mind using this style i will add a check to eslint?

Reply:
So this is the stroustrup brace style? Everything sound complicated for js. :rofl: Yes, I like it (looks a bit like python :stuck_out_tongue:), let's use it!

Reply:
haha 😝